### PR TITLE
pdksync - (FM-8581) - Debian 10 added to travis and provision file refactored

### PR DESCRIPTION
**Pull request by david22swan**
(FM-8581) - Debian 10 added to travis and provision file refactored
pdk version: `1.15.0`

### CODE REVIEW

File: .sync.yml [hunk 1]
```diff
@@ -42,12 +42,38 @@
     services: docker
     sudo: required
     stage: acceptance
+  - bundler_args: 
+    dist: trusty
+    env: PLATFORMS=ub_puppet5
+    rvm: 2.5.3
+    before_script:
+    - bundle exec rake 'litmus:provision_list[waffle_ub]'
```
Comment (lionce) [inline .sync.yml]: Hey @david22swan ,

This should be updated to travis_ub.

Cheers!

File: .sync.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@
     stage: acceptance
   - bundler_args: 
     dist: trusty
-    env: PLATFORMS=debian_puppet6
+    env: PLATFORMS=deb_puppet6
     rvm: 2.5.3
     before_script:
     - bundle exec rake 'litmus:provision_list[waffle_deb]'
```
Comment (lionce) [inline .sync.yml]: Hey @david22swan ,

This should be updated to travis_deb.

Cheers!

File: .sync.yml [hunk 3]
```diff
@@ -18,7 +18,7 @@
   includes:
   - bundler_args: 
     dist: trusty
-    env: PLATFORMS=debian_puppet5
+    env: PLATFORMS=deb_puppet5
     rvm: 2.5.3
     before_script:
     - bundle exec rake 'litmus:provision_list[waffle_deb]'
```
Comment (lionce) [inline .sync.yml]: Hey @david22swan ,

This should be updated to travis_deb.

Cheers!

File: .sync.yml [hunk 4]
```diff
@@ -42,12 +42,38 @@
...
+    - bundle exec rake litmus:acceptance:parallel
+    services: docker
+    sudo: required
+    stage: acceptance
+  - bundler_args: 
+    dist: trusty
+    env: PLATFORMS=ub_puppet6
+    rvm: 2.5.3
+    before_script:
+    - bundle exec rake 'litmus:provision_list[waffle_ub]'
```
Comment (lionce) [inline .sync.yml]: Hey @david22swan ,

This should be updated to travis_ub.

Cheers!

File: .sync.yml (hunk 1 above)
Comment (david22swan) [reply .sync.yml]: fixed

File: .sync.yml (hunk 2 above)
Comment (david22swan) [reply .sync.yml]: fixed

File: .sync.yml (hunk 3 above)
Comment (david22swan) [reply .sync.yml]: fixed

File: .sync.yml (hunk 4 above)
Comment (david22swan) [reply .sync.yml]: fixed

**Pull request merged**